go.mod

```diff
@@ -242,5 +241,3 @@ require (
 )
 
 replace github.com/google/flatbuffers => github.com/google/flatbuffers v1.11.0
-
-replace github.com/dgraph-io/ristretto => github.com/dgraph-io/ristretto v0.1.1-0.20211108053508-297c39e6640f
```

Comment:
Not sure what this line is about, 

I've upgraded `badgerdb` to the latest release in this PR - hence we must remove it for code to compile (or at least adjust somehow).